### ISSUE TITLE
useTimeout should run immediately if `delay` is 0

### ISSUE THREAD

**Issue by oluckyman** [juliencrn/usehooks-ts#65]
`setTimeout(callback, 0)` will call callback immediately. 
I assume that `useTimeout(callback, 0)` should work the same way.
But it doesn't:
```
useEffect(() => {
  // Don't schedule if no delay is specified.
  if (!delay) {
    return
  }
  ...
})
```
https://github.com/juliencrn/usehooks-ts/blob/master/lib/src/useTimeout/useTimeout.ts#L13-L16

**Comment by juliencrn** [comment]
Hi @oluckyman,
Yes, indeed, good catch!
I will fix it now, thank you!

**Comment by juliencrn** [comment]
Fixed in 2.0.2, thanks again! 👍
